### PR TITLE
Make audit recognise hed-only and devel-only in all taps

### PR DESCRIPTION
**Pull request by rkrug**
- add wildcard into expressions to identify head-only and dev-only taps

### CODE REVIEW

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -375,11 +375,11 @@ def audit_homepage
   end
 
   def audit_specs
-    if head_only?(formula) && formula.tap.downcase != "homebrew/homebrew-head-only"
+    if head_only?(formula) && formula.tap.downcase != "*/homebrew-head-only"
```
Comment (jacknagel) [inline Library/Homebrew/cmd/audit.rb]: This won't work... this is a literal string comparison, not a wildcard pattern.

**Comment by DomT4** [comment]
In addition to Jack's comment there's a merge conflict here as well apparently.

File: Library/Homebrew/cmd/audit.rb (hunk 1 above)
Comment (rkrug) [inline Library/Homebrew/cmd/audit.rb]: Sorry - how can I make this work? How can I compare in ruby with wildcard pattern?

**Comment by rkrug** [comment]
I'll supply an updated one later in a new pull request.

**Pull request closed without merging**